composer/trainer/trainer.py

```diff
@@ -782,9 +782,9 @@ def __init__(
         # Save Checkpoint
         save_folder: Optional[str] = None,
         save_filename: str = 'ep{epoch}-ba{batch}-rank{rank}.pt',
-        save_artifact_name: str = '{run_name}/checkpoints/ep{epoch}-ba{batch}-rank{rank}',
+        save_remote_file_name: str = '{run_name}/checkpoints/ep{epoch}-ba{batch}-rank{rank}',
         save_latest_filename: Optional[str] = 'latest-rank{rank}.pt',
-        save_latest_artifact_name: Optional[str] = '{run_name}/checkpoints/latest-rank{rank}',
+        save_latest_remote_file_name: Optional[str] = '{run_name}/checkpoints/latest-rank{rank}',
```

Comment:
do we want to backwards compatibility support this? CC: @bandish-shah 

I'm fine with hard break tbh because all this will go away anyways?

Reply:
Evan is removing those args in his checkpointing refactor anyway

Reply:
Sounds good :)